os_migrate/playbooks/export_networks.yml

```diff
@@ -1,3 +1,6 @@
 - hosts: migrator
+  environment:
+    OS_CLIENT_CONFIG_FILE: "{{ os_migrate_data_dir }}/clouds.yaml"
+    OS_CLOUD: "dst"
```

Comment:
We shouldn't specify OS_CLOUD here, as they may be playbooks that use both src and dst. Rather we should hard-code `cloud: src` wherever we used `os_migrate_src_auth` previously, and respectively for `dst`.

Reply:
We specify OS_CLIENT_CONFIG_FILE which points the OpenStack SDK at the file which defines src and dst variables, and then each module call will have hard-coded either `cloud: src` or `cloud: dst` specifying which entry in the file should be used. IIUC, the OS_CLIENT_CONFIG_FILE variable would be global for the whole Ansible play, not just a role.